src/ring_theory/adjoin/basic.lean

```diff
@@ -54,27 +54,35 @@ variables (R A)
 show adjoin R ⊥ = ⊥, by { apply galois_connection.l_bot, exact algebra.gc }
 
 variables (R) {A} (s)
-theorem adjoin_eq_span : (adjoin R s).to_submodule = span R (monoid.closure s) :=
+lemma submonoid.closure_subset_adjoin : (submonoid.closure s : set A) ⊆ adjoin R s :=
+begin
+  intros x hx,
+  rw [set_like.mem_coe] at hx ⊢,
+  exact submonoid.closure_induction hx (λ x hx, subset_adjoin hx) (one_mem _)
+    (λ x y hx hy, mul_mem _ hx hy),
+end
```

Comment:
Couldn't you state the stronger version that is not specific to `adjoin` (like the old `monoid.closure_subset`), and then pass `subset_adjoin` as an argument when you call this?

Reply:
Indeed we already have `submonoid.closure_le`, so my `submonoid.closure_subset_adjoin` is useless. I modified the proof and deleted `submonoid.closure_subset_adjoin`.